.github/workflows/e2e_test.yml

```diff
@@ -0,0 +1,30 @@
+name: e2e_test
+on:
+  schedule:
+    # Run every night.
```

Comment:
We should document why we chose every night vs every X hours. Also choosing midnight UTC is bad because it creates pressure points on the infra at particular points of the day. I would choose some random time early in the morning during our work day. Like 8:16AM for instance.